src/baseline_bots/bots/smart_order_accepter_bot.py

```diff
@@ -79,6 +83,36 @@ def gen_proposal_reply(self, best_proposer: str, prp_orders: dict, messages: Mes
                     proposer, str(msg)
                 )
         return messages
+    
+    def respond_to_invalid_orders(self, invalid_proposal_orders: Dict[str, List[str]], messages_data: MessagesData) -> None:
+        """
+        The bot responds by HUHing the invalid proposal orders received (this could occur if the move proposed is invalid)
+
+        :param invalid_proposal_orders: dictionary of sender -> invalid orders proposed
+        :param messages_data: Message Data object to add messages
+        """
+        if not invalid_proposal_orders:
+            return
+        for sender in invalid_proposal_orders:
+            message = HUH(PRP(ORR(XDO(dipnet_to_daide_parsing(invalid_proposal_orders[sender])))))
+            messages_data.add_message(
+                sender, str(message)
+            )
+        
+    def respond_to_alliance_messages(self, messages_data: MessagesData) -> None:
```

Comment:
We always say YES to any ALY? what if enemies (negative stance) propose?

Reply:
Right now, I just stuck to what Denis proposed. I could change that after discussing in the meeting.

Reply:
ok thanks